client/src/components/Table.jsx

```diff
@@ -22,38 +47,87 @@ const LogTable = props => {
-                <TableCell>{observation.physical.feverSeverity}</TableCell>
-                <TableCell>{observation.physical.chillsSeverity}</TableCell>
-                <TableCell>{observation.physical.shortnessOfBreathSeverity}</TableCell>
-                <TableCell>{observation.physical.soreThroatSeverity}</TableCell>
-                <TableCell>{observation.physical.chestPainSeverity}</TableCell>
-                <TableCell>{observation.physical.fatigueSeverity}</TableCell>
-                <TableCell>{observation.physical.bluishnessSeverity}</TableCell>
-                <TableCell>{observation.nonPhysical.openComment}</TableCell>
+                {questions.map(question => (
+                  <TableCell key={question}>{question}</TableCell>
+                ))}
               </TableRow>
-            </TableBody>
-          ))}
-        </Table>
-      </TableContainer>
-      <Chart chartType={chartType.bar} />
-    </div>
+            </TableHead>
+            {(detailData.length ? detailData : observations).map(observation => (
+              <TableBody key={observation.date}>
+                <TableRow>
+                  <TableCell>{new Date(observation.date).toLocaleDateString()}</TableCell>
+                  <TableCell>{observation.physical.dailyfeeling}/5</TableCell>
+                  <TableCell>{observation.physical.coughSeverity}</TableCell>
+                  <TableCell>{observation.physical.feverSeverity}</TableCell>
+                  <TableCell>{observation.physical.chillsSeverity}</TableCell>
+                  <TableCell>{observation.physical.shortnessOfBreathSeverity}</TableCell>
+                  <TableCell>{observation.physical.soreThroatSeverity}</TableCell>
+                  <TableCell>{observation.physical.chestPainSeverity}</TableCell>
+                  <TableCell>{observation.physical.fatigueSeverity}</TableCell>
+                  <TableCell>{observation.physical.bluishnessSeverity}</TableCell>
+                  <TableCell>{observation.nonPhysical.openComment}</TableCell>
+                </TableRow>
+              </TableBody>
+            ))}
+          </Table>
+        </TableContainer>
+        <div>
+          <FormControlLabel
+            control={<Checkbox checked={feverClicked} onChange={handleChange} color="primary" name="Fever" />}
+            label="Fever"
+          />
+          <FormControlLabel
+            control={<Checkbox checked={behaveClicked} onChange={handleChange} name="Behavioral" color="primary" />}
+            label="Behavioral"
+          />
+        </div>
+      </div>
+      {behaveClicked ? <Typography variant="h2">Your Behavioral Health Progress</Typography> : null}
+      {renderBehavior}
+      {behaveClicked ? (
+        <div className={classes.tagLine}>
+          <Typography variant="h6">
+            See this past week's daily progress. Hover over the chart to see individual scores.
+          </Typography>
+        </div>
```

Comment:
Let's remove the chart from the UI at least for this PR - I will open up another issue capturing implementing it. Let's just capture the scope of adding behavioral elements to CoronaTracker for this PR

Reply:
for Table.jsx? it would be really difficult to remove the entire component because then checkboxes are integrated with it too.

Reply:
I wouldn't remove the entire component, just remove it from the UI - you can use empty `<div>`s  etc.

This PR is getting much too large so we need to merge in a minimum viable solution here and then scope out more pointed issues to implement the rest from here.

Reply:
I do agree this PR is getting bigger than previously anticipated.  To clarify, we should remove the checkboxes (fever & behavioral chart) and the behavioral chart from Table.jsx?

Reply:
If possible, yes. They're great features to have and I'd love to to get them merged in, just in a different PR so we can keep scope a little clearer :)

Reply:
Im also working on table.jsx for issue #370 , so it might be a good idea to cut it out of scope for 370 when you make a new issue for it

Reply:
Removed the checkboxes and chart. Hopefully, this is good to go now!